.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
```

Comment:
could you please include the.settings and .classpath in the .gitignore file and also stop tracking them as well